.jenkins/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
```

Comment:
This seems to be a leftover file you should delete.

Reply:
Done. File removed